diagnostic.md

```diff
@@ -91,9 +118,8 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+color: yellow;
+font-family: sans-serif;
```

Comment:
There is inline styling the takes precedent over the stylesheet specifying that the font-family will be cursive